hardware/HW-Boot-Camp.md

```diff
@@ -0,0 +1,22 @@
+********
+![hardware](files/hardware.png)
+********
+
+# Hardware Boot Camp
+
+## One Day Workshop
+| Time | Content |  
+|:---- |:----|    
+| 1 hour | [Wireless-Products-Protocols-and-Applications](HW-Wireless-Products-Protocols-and-Applications) |
+| 2 hour | [Schematic-Design-Guide](HW-Schematic-Design-Guide) |
+| 3 hour | [PCB-Layout-Guide](HW-PCB-Layout-Guide) |
+| 1.5 hour | [Comply-to-the-Regulations(Cerfifications)](HW-Comply-to-the-Regulations-Cerfifications )|
```

Comment:
HW-Comply-to-the-Regulations-Cerfifications -> Comply-to-the-Regulations-Cerfifications